magpie-persist/src/main/java/io/openraven/magpie/plugins/persist/PersistPlugin.java

```diff
@@ -38,7 +38,6 @@ public void accept(MagpieEnvelope env) {
       AssetModel asset = MAPPER.map(env);
       assetsRepo.upsert(asset);
       logger.info("Saved asset with id: {}", asset.getAssetId());
-      System.out.println("SOUT - Saved asset with id: " + asset.getAssetId());
```

Comment:
Ah damn, I hope I wasn't the one who added that line!

Reply:
it was mine introduced 21.08
forgot to cleanup this after testing and investigation